### PR TITLE
19003 Feature-vue3-upgrade-fee-services

### PR DESCRIPTION
**Pull request by jamespaologarcia**
*Issue #:* bcgov/entity#19003

Fee services upgrade to vue3

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/fee-summary/FeeSummary.vue [hunk 1]
```diff
@@ -72,66 +72,66 @@ import SbcFeeSummary from './SbcFeeSummary.vue'
 export default class FeeSummary extends Vue {
   readonly FeeSummaryActions = FeeSummaryActions
 
-  /** Filing information to calculate fees. */
-  @Prop({ default: () => [] }) readonly filingData!: Array<FilingDataIF>
+/** Filing information to calculate fees. */
```
Comment (severinbeauvais) [inline src/components/fee-summary/FeeSummary.vue:75]: This (and many below) should be indented, since they are in a `{` block.

**Review by severinbeauvais — APPROVED** [review]
Please fix indents first.

File: src/components/fee-summary/FeeSummary.vue [hunk 2]
```diff
@@ -105,33 +105,33 @@ export default class FeeSummary extends Vue {
 
 #fee-summary {
   .v-btn {
-    width: 100%;
-    border: 1.25px solid $app-blue;
-    color: $app-blue;
-    box-shadow: 0 1px 2px 0 rgba(33,37,41,0.2);
-    border-radius: 4px;
+      width: 100%;
```
Comment (severinbeauvais) [inline src/components/fee-summary/FeeSummary.vue:108]: Now you have too much indentation here.

2 spaces per indent, please.

**Comment by ketaki-deodhar** [comment]
You need to change `package.json` file accordingly as well with latest dependency version if any

**Comment by jamespaologarcia** [comment]
> You need to change `package.json` file accordingly as well with latest dependency version if any

On it...

**Pull request closed without merging**